controllers/permits/applications/special-uses/noncommercial/index.js

```diff
@@ -58,14 +57,26 @@ get.id = function(req, res){
 		noncommercialFields.endDateTime = '2017-04-15 20:00:00';
 		noncommercialFields.numberParticipants = 45;
 
-		util.copyGenericInfo(cnData, jsonData);
-		jsonData.noncommercialFields = noncommercialFields;    
-		jsonResponse.success = true;
-        
+		jsonData = util.copyGenericInfo(cnData, jsonData);
```

Comment:
Should only run if application exists. Okay for now, but when unique data is retrieved from DB this should be moved inside the callback for getting application.